### ISSUE TITLE
BucketSinging error

### ISSUE THREAD

**Issue by Raemon** [dspeyer/ritualEngine#113]
check@https://secularsolstice2020.org/widgets/BucketSinging/lib.js:20:11
concat_chunks@https://secularsolstice2020.org/widgets/BucketSinging/audiochunk.js:215:18
handle_message@https://secularsolstice2020.org/widgets/BucketSinging/app.js:847:32
handle_message@https://secularsolstice2020.org/widgets/BucketSinging/app.js:527:10

Blocked by: gwillen/solstice-audio-test#98

**Comment by taymonbeal** [comment]
Duplicate of #103.
